<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="1701" started-at="2019-02-08T10:08:54Z" finished-at="2019-02-08T10:08:56Z">
    <groups>
    </groups>
    <test name="Regression1" duration-ms="1701" started-at="2019-02-08T10:08:54Z" finished-at="2019-02-08T10:08:56Z">
      <class name="com.nagarro.test.TestCaseClass">
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.nagarro.test.TestCaseClass@2c6a3f77]" name="beforeMethod" is-config="true" duration-ms="15" started-at="2019-02-08T15:38:54Z" finished-at="2019-02-08T15:38:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.nagarro.test.TestCaseClass.testcase1()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testcase1()[pri:0, instance:com.nagarro.test.TestCaseClass@2c6a3f77]" name="testcase1" duration-ms="1655" started-at="2019-02-08T15:38:54Z" description="Priority:high$$sevirity:High$$Test:2" finished-at="2019-02-08T15:38:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [23] but found [12]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [23] but found [12]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at myProject.AsserUtils.assertEquals(AsserUtils.java:14)at com.nagarro.test.TestCaseClass.testcase1(TestCaseClass.java:34)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="testcase2()[pri:0, instance:com.nagarro.test.TestCaseClass@2c6a3f77]" name="testcase2" duration-ms="0" started-at="2019-02-08T15:38:56Z" description="Priority:high$$sevirity:High$$test:1" finished-at="2019-02-08T15:38:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase2 -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.nagarro.test.TestCaseClass@2c6a3f77]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2019-02-08T15:38:56Z" finished-at="2019-02-08T15:38:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.nagarro.test.TestCaseClass.testcase2()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
      </class> <!-- com.nagarro.test.TestCaseClass -->
    </test> <!-- Regression1 -->
  </suite> <!-- Suite1 -->
</testng-results>
